diff --git a/modules/games/Makefile.am b/modules/games/Makefile.am
index 961bc37..c688f01 100644
--- a/modules/games/Makefile.am
+++ b/modules/games/Makefile.am
@@ -1 +1 @@
-SUBDIRS = mtsec minisec rfts risk tae
+SUBDIRS = minisec risk 
diff --git a/modules/tpcl/guile/tpguile.cpp b/modules/tpcl/guile/tpguile.cpp
index 58a9fec..ab07569 100644
--- a/modules/tpcl/guile/tpguile.cpp
+++ b/modules/tpcl/guile/tpguile.cpp
@@ -435,6 +435,6 @@ TpGuile::TpGuile() {
     std::string  designTypeDefFile = "guile.scm";
 
     scm_init_guile();
-    scm_c_eval_string("(set! %load-path (cons \"" DATADIR "/tpserver/tpscheme/guile/\" (cons \"modules/tpcl/guile/\" (cons \"../modules/tpcl/guile/\" %load-path ))))");
+    scm_c_eval_string("(set! %load-path (cons \".\" (cons \"" DATADIR "/tpserver/tpscheme/guile/\" (cons \"modules/tpcl/guile/\" (cons \"../modules/tpcl/guile/\" %load-path )))))");
     scm_c_primitive_load_path( designTypeDefFile.c_str());
 }
diff --git a/Makefile.am b/Makefile.am
index 1c32bba..f5fa508 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = libltdl modules tpserver
+SUBDIRS = modules tpserver
 
 man_MANS = tpserver-cpp.1
 
diff --git a/tpserver/Makefile.am b/tpserver/Makefile.am
index a115a8f..a58323b 100644
--- a/tpserver/Makefile.am
+++ b/tpserver/Makefile.am
@@ -11,13 +11,8 @@ GNUTLSSOURCES =
 GNUTLSHEADERS =
 endif
 
-if AVAHI
-AVAHISOURCES = avahi.cpp
-AVAHIHEADERS = avahi.h
-else
 AVAHISOURCES =
 AVAHIHEADERS =
-endif
 
 tpserver_cpp_SOURCES = $(GNUTLSSOURCES) main.cpp connection.cpp \
 		frame.cpp game.cpp logging.cpp net.cpp settings.cpp player.cpp object.cpp \
